CommandDotNet.Tests/ModelTests.cs

```diff
@@ -37,70 +37,70 @@ public void CanConstructorsReadModels()
+                    person,
+                    time,
+                    address
+                };
             
-            string content = JsonConvert.SerializeObject(obj, Formatting.Indented, new JsonSerializerSettings
-            {
-                ContractResolver = new CamelCasePropertyNamesContractResolver(),
-            });
+                string content = JsonConvert.SerializeObject(obj, Formatting.Indented, new JsonSerializerSettings
+                {
+                    ContractResolver = new CamelCasePropertyNamesContractResolver(),
+                });
             
-            File.WriteAllText("TestCases/ModelTests.ModelTests.Output.json", content);
-            return 5;
+                File.WriteAllText("TestCases/ModelTests.ModelTests.Output.json", content);
+                return 5;
+            }
         }
-    }
-
-    public class ModelAppWithConstructor
-    {
-        private readonly Person _person;
 
-        public ModelAppWithConstructor(Person person)
+        public class ModelAppWithConstructor
         {
-            _person = person;
-        }
+            private readonly Person _person;
 
-        public int ReadPersonData()
-        {
-            if (_person != null && _person.Id == 9)
-                return 6;
-            return 1;
+            public ModelAppWithConstructor(Person person)
+            {
+                _person = person;
+            }
+
+            public int ReadPersonData()
+            {
+                if (_person != null && _person.Id == 9)
+                    return 6;
+                return 1;
+            }
         }
-    }
     
 
-    public class Person: IArgumentModel
-    {
-        public int Id { get; set; }
-        public string Name { get; set; }
-        [Option]
-        public string Email { get; set; }
+        public class Person: IArgumentModel
+        {
+            public int Id { get; set; }
+            public string Name { get; set; }
+            [Option]
+            public string Email { get; set; }
         
-        public string Number { get; set; }
-        public Job Job { get; set; }
-    }
+            public string Number { get; set; }
+            public Job Job { get; set; }
+        }
 
-    public class Address : IArgumentModel
-    {
-        public string City { get; set; }
+        public class Address : IArgumentModel
+        {
+            public string City { get; set; }
         
-        [Option(ShortName = "a")]
-        public bool HasAirport { get; set; }
-    }
+            [Option(ShortName = "a")]
+            public bool HasAirport { get; set; }
+        }
 
-    public class Job : IArgumentModel
-    {
-        public string Company { get; set; }
-        [Option]
-        public string Position { get; set; }
+        public class Job : IArgumentModel
+        {
+            public string Company { get; set; }
+            [Option]
+            public string Position { get; set; }
+        }
```

Comment:
moved these types into the test type so they don't conflict with types of the same name else where.  since they're only used for the test, nesting them in the test is a good way to isolate them.